src/Microsoft.AspNetCore.Mvc.Abstractions/Formatters/IFormatterExceptionPolicy.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.Formatters
+{
+    /// <summary>
+    /// A policy which <see cref="IInputFormatter"/>s can implement to indicate if they want the body model binder
+    /// to convert all exceptions to model state errors. By default, all in-built <see cref="IInputFormatter"/>s
+    /// implement this interface and have a default value of <see langword="true"/>.
+    /// </summary>
+    public interface IFormatterExceptionPolicy
+    {
+        bool HandlesExceptions { get; }
```

Comment:
I feel this name to be confusing as the input formatters are merely saying if they honor the setting on MvcOptions and we wanted a way for derived formatters to opt out of it. Any suggestions here? cc @Eilon

Reply:
This is not "HandlesExceptions". The meaning of this is "should my caller handle **all exceptions**". 

Maybe we should call it `CallerMustHandleExceptions`?

Also @Eilon should we `[Obsolete]` this now since we're planning to remove this in 3.0?

Reply:
In view of how we wanted to make the option sound 'false' in MvcOptions, I have modified it to sound the same here `CallerMustNotHandleAllExceptions`